packages/ncids-css/scss/components/nci-hero-cta/internal/_nci-hero-cta.scss

```diff
@@ -0,0 +1,5 @@
+.nci-hero-cta{
```

Comment:
can we rename this since there is a `.nci-hero__cta` class in the nci-hero? maybe `.nci-hero--with-cta-strip` or something? i don't really feel strongly about the name apart from "something else"

not really a big deal, just a thought

Reply:
we should discuss this